backend/LoadCell.py

```diff
@@ -0,0 +1,104 @@
+from enum import Enum
+import matplotlib.pyplot as plt
+import json
+import numpy as np
+
+class CalLoadCell():
+    class State(Enum):
+        NORMAL = 0
+        STORE = 1
+        WAIT = 2
+
+    def __init__(self, name="DefaultLoadCell") -> None:
+        self.name = name
+        self.load_cell = 0
+        self.tare = 0
+        self.slope = 1
+        self.vol_offset = 0
+        self.cal = False
+        self.cal_points = {}
+        self.mass = 0
+        self.volts = 0
+        self.txtval = 0
+        self.current_state = self.State.NORMAL
```

Comment:
Also if you are storing the data in a file locally then this needs to be loaded on construction, as a LoadCell object should not exist without it's cal if it was already cal'd at one point in time